addon/templates/components/frost-modal-dialog.hbs

```diff
@@ -1,44 +1,64 @@
-<div class="overlay">
-  <div class="modal">
+{{#ember-remodal
+  forService=true
+  closeOnOutsideClick=false
+  onOpen=(action 'modalOpen')
+  onClose=(action 'modalClose')
+  onConfirm=(action 'confirm')
+  closeOnEscape=false
+  disableNativeClose=true as |modal|
+}}
+  {{yield}}
+  {{#modal.open}}
+    {{#yield-slot 'target'}}
+      {{yield}}
+    {{/yield-slot}}
+  {{/modal.open}}
+
+  {{#yield-slot 'header'}}
+    {{yield}}
+  {{else}}
     <div class="header">
       {{title}}
     </div>
+  {{/yield-slot}}
+
+  {{#yield-slot 'body'}}
     <div class="body">
       {{frost-icon
         icon=dialogIcon
         class='icon'}}
       {{#if messageTemplate}}
         {{partial messageTemplate}}
       {{else}}
-        <span>{{message}}</span>
-      {{/if}}
-    </div>
-    <div class="footer">
-      {{#if actionsTemplate}}
-        {{partial actionsTemplate}}
-      {{else}}
-        {{#if confirmAlias}}
-          {{frost-button
-            autofocus=true
-            onClick=(action "cancel")
-            size="medium"
-            priority="tertiary"
-            text=cancelAlias
-          }}
-          {{frost-button
-            onClick=(action "confirm")
-            size="medium"
-            priority="primary"
-            text=confirmAlias}}
-        {{else}}
-          {{frost-button
-            autofocus=true
-            onClick=(action "cancel")
-            size="medium"
-            priority="primary"
-            text=cancelAlias}}
-        {{/if}}
+        <span>{{yield}}</span>
       {{/if}}
     </div>
+  {{/yield-slot}}
+
+  <div class="footer">
+    {{#if confirmAlias}}
+      {{#modal.cancel}}
+        {{frost-button
+          autofocus=true
+          size="medium"
```

Comment:
Are you wanting to use single quotes here for the attribute? size='medium'

I saw other instances for this below as well.